packages/core/admin/admin/src/translations/en.json

```diff
@@ -680,6 +680,8 @@
   "components.popUpWarning.message": "Are you sure you want to delete this?",
   "components.popUpWarning.title": "Please confirm",
   "content-manager.App.schemas.data-loaded": "The schemas have been successfully loaded",
+  "content-manager.actions.clone.error": "An error occurred while trying to clone the document.",
```

Comment:
Is it weird that in one place we say 'duplicate' and somewhere else say 'clone'?